src/probnum/diffeq/odefiltsmooth/kalman_odesolution.py

```diff
@@ -40,38 +40,43 @@ class KalmanODESolution(ODESolution):
     --------
     >>> from probnum.diffeq import logistic, probsolve_ivp
     >>> from probnum import random_variables as rvs
-    >>> initrv = rvs.Constant(0.15)
-    >>> ivp = logistic(timespan=[0., 1.5], initrv=initrv, params=(4, 1))
-    >>> solution = probsolve_ivp(ivp, method="ekf0", step=0.1)
+    >>>
+    >>> def f(t, x):
+    ...     return 4*x*(1-x)
+    >>>
+    >>> y0 = np.array([0.15])
+    >>> t0, tmax = 0., 1.5
+    >>> solution = probsolve_ivp(f, t0, tmax, y0, step=0.1, atol=None, rtol=None)
```

Comment:
this is recovering the previous doctest (atol and rtol at None implies fixed steps). 
I am not sure whether we might want an adaptive step example, but what is the doctest here (the outputs might change marginally depending on some default tolerances which I guess are subject to change in the next weeks)

Reply:
Just to question it: Is there an alternative syntax for choosing fixed steps? It seems inconvenient to set two values to `None`, maybe just checking `step is not None` is enough? Or something more explicit, like `adaptive=false`?

Reply:
Let's talk about this later today.

Reply:
Decision: include `adaptive=True/False`